### DISCUSSION TITLE
How to pass the validation message to a custom form field component ?

### DISCUSSION THREAD

**Discussion by pleky2** [nylo-core/nylo#170] [Q&A]
form_cast.dart
![image](https://github.com/user-attachments/assets/208b129b-f0c1-47cb-aa58-aef0339d59ef)

this is my form.
![image](https://github.com/user-attachments/assets/b1b61f47-681b-4f37-ab91-87ceab31ec60)

I am using this code to validate the form
`  form.submit(
      onSuccess: (value) {
        print(value);
      },
      onFailure: (exception) {}, );`
      
When i print the validation message is always return null but i still can got the error message from the toast.
![image](https://github.com/user-attachments/assets/0666055e-159a-49ad-bb44-49053422dc9d)

**Comment by agordn52** [comment]
Hi @pleky2,

Sorry for the late reply. 

Which version of Nylo are you using?
